### ISSUE TITLE
Move camera back [Feature request] 

### ISSUE THREAD

**Issue by tstandley** [leoetlino/project-restoration#111]
I guess this is a QOL improvement, but when playing on a high resolution monitor, it'd be nice to be able to move the camera back further to see more of the environment. Increasing the draw distance would be nice too.

**Comment by Luigimeansme** [comment]
Draw distance improval would be something only doable on emulator. On console, you would just have ugly draw distance

**Comment by leoetlino** [comment]
Sounds like a good idea to me. Months ago I considered making the game nicer to play on large screens by doing the FOV edit you suggested and editing a bunch of UI layouts, but sadly I didn't have enough free time to work on it.

This probably belongs in a separate (Project Restoration-based) project, though.

**Comment by tstandley** [comment]
Yeah, probably. You wouldn't want to have an exponential number of versions for each feature you want to enable.
